src/extension/importer/core/src/main/java/org/geoserver/importer/FileData.java

```diff
@@ -28,7 +29,7 @@
     protected Resource file;
 
     public FileData(Resource file) {
-        this.file = file;
+        this.file = Resources.serializable(file);
```

Comment:
As I suggested, I made a serializable resource wrapper for the sake of convenience. This way we hardly need to change any code in the other modules... seemed like a good idea to me, no?

As far as @cuttlefish is concerned, I believe I can help him through a configuration change.